creator-node/src/middlewares.js

```diff
@@ -762,14 +762,21 @@ async function getReplicaSetSpIDs({
 
       try {
         // will throw error if blocknumber not found
-        replicaSet =
-          await libs.contracts.UserReplicaSetManagerClient.getUserReplicaSetAtBlockNumber(
-            userId,
-            blockNumber
-          )
-        errorMsg = null
-        blockNumberIndexed = true
-        break
+        const encodedUserId = libs.Utils.encodeHashId(userId)
+        const spResponse = await libs.discoveryProvider.getUserReplicaSet({
```

Comment:
haven't read through API but what will this do if blocknumber not indexed? will it retry internally? or error

Reply:
also since this is a new function, i assume it only worked bc libs are linked locally
would try it without that, and we'll prob need to get CN to consume latest sdk version

Reply:
the current implementation will return null if the blocknumber is not yet indexed